aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/renderers/CommentsList.get.js

```diff
@@ -19,6 +19,7 @@ model.jsonModel = {
          name: "alfresco/renderers/CommentsList",
          id: "COMMENT_LIST",
          config: {
+            addCommentsFullScreen: true,
```

Comment:
No, I appreciate that this is the test WebScript - however it would be useful for the test WebScript to represent the default as well as optional configuration parameters (to ensure that both modes work adequately). There is an iframe widget that was added for the analytics work (alfresco/integration/IFrame) that we might be able to make use of here (given that this is the exact use case that we're trying to solve).